default.nix

```diff
@@ -1,7 +1,8 @@
-
-{ pkgs ? import <nixpkgs> {} }:
+{ nurpkgs ? import <nixpkgs> {} # For nixpkgs dependencies used by NUR itself
+, pkgs ? import <nixpkgs> {} # Dependencies to call NUR repos with
```

Comment:
Should we make this argument mandatory and add a error message if not supplied by the user?

Reply:
The error message could then also include the canonical syntax to import NUR.

Reply:
Not sure, making the `pkgs` argument mandatory would mean that `import <nur> {}` wouldn't work, and you couldn't evaluate it anymore directly, e.g. `nix-instantiate '<nur>' -A repos.infinisil.foo` wouldn't work anymore.

Reply:
Should it then default to one `import` expression for example:

```
let fallback = import <nixpkgs> {} in {
  nurpkgs ?  fallback,
  pkgs ? fallback
}
```

I am not sure if this make any difference in terms of performance.

Reply:
According to @cleverca22 and @LnL7 in #nixos, what you proposed should be faster, so I'll change it to that.